examples/with-cloud/app/MyRuntimeProvider.tsx

```
@@ -0,0 +1,43 @@
+const useThreadRuntime = () => {
+  const runtime = useEdgeRuntime({
+    api: "/api/chat",
+    unstable_AISDKInterop: true,
+    unstable_sendMessageIds: true,
+  });
+
+  return runtime;
+};
+
+const cloud = new AssistantCloud({
+  baseUrl: process.env["NEXT_PUBLIC_ASSISTANT_BASE_URL"]!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)